src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,116 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types/Person';
+import { SearchParams } from '../utils/searchHelper';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  persons: Person[];
+};
+export const PeopleTable: React.FC<Props> = ({ persons }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const rowData = {
+    Name: 'name',
+    Sex: 'sex',
+    Born: 'born',
+    Died: 'died',
+  };
+
+  const setSorting = (value: string): SearchParams => {
+    if ((!sort && !order) || (sort !== value)) {
+      return { sort: value };
+    }
+
+    if (sort === value && !order) {
+      return { sort: value, order: 'desc' };
+    }
+
+    return { sort: null, order: null };
+  };
+
+  const sortPeople = persons.sort((first, second) => {
+    switch (sort) {
+      case 'name':
```

Comment:
[Try not to hardcode these values, but use ](https://www.typescriptlang.org/docs/handbook/enums.html)[Enum](https://www.typescriptlang.org/docs/handbook/enums.html) instead